server/routes/induction/update/additionalTrainingUpdateController.ts

```diff
@@ -20,7 +20,7 @@ export default class AdditionalTrainingUpdateController extends AdditionalTraini
     next: NextFunction,
   ): Promise<void> => {
     const { prisonNumber, journeyId } = req.params
-    const { inductionDto } = req.session
+    const { inductionDto } = req.journeyData
```

Comment:
The changes to all the Update controllers is basically the same ...  get the `InductionDto` from journeyData instead of the session; and either put it back on journeyData rather than the session, or clear the journeyData (depending on the flow in the existing code)